Bio/Seq.py

```diff
@@ -28,7 +28,8 @@
 from Bio import BiopythonWarning
 from Bio import Alphabet
 from Bio.Alphabet import IUPAC
-from Bio.Data.IUPACData import ambiguous_dna_complement, ambiguous_rna_complement
+from Bio.Data.IUPACData import (ambiguous_dna_complement,
+                                ambiguous_rna_complement)
```

Comment:
Is this a PEP8 change? The brackets are something I don't recall seeing elsewhere in the Biopython codebase?